runtime/common/src/lib.rs

```diff
@@ -116,12 +116,6 @@ pub mod proxy {
 		NonTransfer,
 		/// Proxy with the ability to reject time-delay proxy announcements.
 		CancelProxy,
-		/// Assets proxy. Can execute any call from `assets`, **including asset transfers**.
-		Assets,
-		/// Owner proxy. Can execute calls related to asset ownership.
-		AssetOwner,
-		/// Asset manager. Can execute calls related to asset management.
-		AssetManager,
```

Comment:
No proxies of these types exist on the running testnet. Nor collator which will change it's index within the enum.